english/pack-lunch/cheese-puzzle.md

```diff
@@ -0,0 +1,2 @@
+[it makes me want to cry...](http://2.bp.blogspot.com/-8VIW5EHOXis/VC6PctbH9kI/AAAAAAAAM0s/n_5JtDdrhQY/s1600/IMG_3434.JPG)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.